packages/db/tests/models/test-granule-model.js

```diff
@@ -49,21 +57,251 @@ test.after.always(async (t) => {
   });
 });
 
-test('GranulePgModel.upsert() creates a new running granule', async (t) => {
+test('getWithExecutionHistory returns execution history for granule with one execution', async (t) => {
```

Comment:
I would appreciate close scrutiny over all of the unit tests in this file

Reply:
The test logic is really good and covers most of the case (and explained a few questions from above),  I'd like to re-review it again after the questions for the granule model are discussed/resolved, but this looks great ,generally.